src/geff/_cli.py

```diff
@@ -1,31 +1,159 @@
 import logging
 import sys
+from pathlib import Path
+from typing import TYPE_CHECKING, Annotated, cast
 
 import typer
 import zarr
 
 from . import utils
 from .metadata_schema import GeffMetadata
 
+if TYPE_CHECKING:
+    from typing import Literal
+
 app = typer.Typer(help="GEFF Command Line Interface")
 
 logging.basicConfig(stream=sys.stderr, level=logging.WARNING, format="%(levelname)s: %(message)s")
 logging.captureWarnings(True)
```

Comment:
i know it's not from this PR... but do you know why we're configuring/using logging here?

Reply:
No clue. Happy to pull it out if it's not useful